galaxy/importer/linters/__init__.py

```diff
@@ -23,6 +23,9 @@
 
 LOG = logging.getLogger(__name__)
 
+LINTERS_DIR = os.path.abspath(os.path.dirname(__file__))
```

Comment:
Would it be worthwhile to make this configurable? 

Don't really need it to be configurable yet though so maybe something for later.